supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-data-grid.adoc

```diff
@@ -84,18 +84,18 @@ For documentation questions, contact jdg-docs@redhat.com.
 [discrete]
 [[red-hat-data-grid]]
 ==== image:images/yes.png[yes] Red Hat Data Grid (noun)
-*Description*: Red Hat Data Grid is a high-performance, distributed in-memory data store. Use "Red Hat Data Grid" in the first instance and "Data Grid" in all subsequent instances. This product name applies to Red Hat Data Grid 7.3 and later versions.
```

Comment:
This isn't the right file to make this change in. Since we did the reorg, all entries are in the appropriate letter, so yours would be in r.adoc.

I actually have a PR open to remove these old product files, for this very reason. I'll ping again to get one last approval to get that PR merged.

Reply:
> Also, I just noticed that the commit message/description for this PR is actually from Laura's OCS->ODF PR. Can you fix that so that the commit message accurately shows what you were changing here?

Hi @bergerhoffer . My apologies, I thought I needed to keep titles somewhat similar given the specific request for these types of GitHub Issues. I updated the title and addressed your other feedback. Please let me know your thoughts.